src/main/resources/assets/vampirism/lang/en_us.json

```diff
@@ -1608,6 +1609,71 @@
   "stat_type.vampirism.action_cooldown_time": "Action On Cooldown",
   "stat_type.vampirism.skill_unlocked": "Skill Unlocked",
   "stat_type.vampirism.skill_forgotten": "Skill Forgotten",
+  "__comment": "vampire books",
+  "vampire_book.vampirism.unknown_id": "Unknown Book",
```

Comment:
I do not feel good about adding the book content in the lang files as discussed previously. Lang files are supposed to contain a simple list of key value pairs to describe things.


They are not a place to store load amount of book content. [Here](https://github.com/neoforged/NeoForge/issues/1519#issuecomment-2337833818) was a little bit a discussion about a similar thing.


Regarding the translation, I doubt that there are a lot of people who want to translate this correctly into the languages and not just use an AI to do this. Then we do not need to put it in a languages file and publish it to crowdin which has word limits.